network/rtp_packet.ksy

```diff
@@ -0,0 +1,85 @@
+meta:
+  id: rtp_packet
+  title: rtp protocol
+  xref:
+    rfc: 3550
+    wikidata: https://en.wikipedia.org/wiki/Real-time_Transport_Protocol
+  license: Unlicense
+  endian: be
+
+doc: |
+  The Real-time Transport Protocol (RTP) is a network protocol for delivering audio and video over IP networks. RTP is used in communication and entertainment systems that involve streaming media, such as telephony, video teleconference applications including WebRTC, television services and web-based push-to-talk features.
+  RTP typically runs over User Datagram Protocol (UDP). RTP is used in conjunction with the RTP Control Protocol (RTCP). While RTP carries the media streams (e.g., audio and video), RTCP is used to monitor transmission statistics and quality of service (QoS) and aids synchronization of multiple streams. RTP is one of the technical foundations of Voice over IP and in this context is often used in conjunction with a signaling protocol such as the Session Initiation Protocol (SIP) which establishes connections across the network.
+  RTP was developed by the Audio-Video Transport Working Group of the Internet Engineering Task Force (IETF) and first published in 1996 as RFC 1889, superseded by RFC 3550 in 2003.
+
```

Comment:
1. This description seems to be copy-pasted from Wikipedia, it's licensed as [https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License](CC BY-SA 3.0 Unported), you either need to mention that in `meta/license`, or rewrite this text to be able to use `Unlicense`.

2. Please don't use such long lines, as they will be literally pasted into docstring comments this way. It's better to wrap it around ~80th column.